### PR TITLE
add support for rails 5

### PR DESCRIPTION
**Pull request by craig-day**
/cc @grosser 

might want to bump to version 1.0 since we can't support both rails 5 and ruby 2.1 or lower.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,6 @@ language: ruby
 bundler_args: ""
 sudo: false
 rvm:
-  - 2.0.0
-  - 2.1.8
```
Comment (grosser) [inline .travis.yml]: I need at least 2.1 for some projects ... can you ad an exclude instead and keep 2.1 ? ...

File: forking_test_runner.gemspec [hunk 2]
```diff
@@ -8,14 +8,14 @@ Gem::Specification.new name, ForkingTestRunner::VERSION do |s|
   s.homepage = "https://github.com/grosser/#{name}"
   s.files = `git ls-files lib/ bin/ MIT-LICENSE`.split("\n")
   s.add_runtime_dependency "parallel_tests", ">= 1.3.7"
-  s.add_runtime_dependency "activerecord", "< 5.0.0"
+  s.add_runtime_dependency "activerecord", "< 6.0.0"
```
Comment (grosser) [inline forking_test_runner.gemspec]: better 5.1 ;)

**Comment by grosser** [comment]
glad to see there was no funky failures :D

**Comment by craig-day** [comment]
alright, brought back 2.0 and 2.1 but told travis to not build with rails 5.0 gemfile there.

**Comment by grosser** [comment]
0.7.1

**Pull request merged**